docs/plugins.rst

```diff
@@ -212,7 +212,7 @@ definitions in Airflow.
         flask_blueprints = [bp]
         appbuilder_views = [v_appbuilder_package]
         appbuilder_menu_items = [appbuilder_mitem]
-        stat_name_handler = stat_name_dummy_handler
+        stat_name_handler = staticmethod(stat_name_dummy_handler)
```

Comment:
Without this, assigning the function in this way binds it to the AirflowPlugin class. Illustration of the difference (on Python2):

```
>>> def x(arg_that_should_be_str):
...   return arg_that_should_be_str
...
>>> class Foo:
...   x = x
...
>>> x_impl = Foo.x
>>> x_impl("a string")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: unbound method x() must be called with Foo instance as first argument (got str instance instead)
>>>
>>> class Foo:
...   x = staticmethod(x)
...
>>> x_impl = Foo.x
>>> x_impl("a string")
'a string'
```

Reply:
hmm interesting

Reply:
```>>> def x(arg_that_should_be_str):
...    return arg_that_should_be_str
...
>>> class Foo:
...    x = x
...
>>> x_impl = Foo.x
>>> type(x_impl)
<type 'instancemethod'>
>>> x_impl("a string")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: unbound method x() must be called with Foo instance as first argument (got str instance instead)
>>> ^D
tfeng@tfeng-mbp143 ~/playground/incubator-airflow (tfeng_fix_ci_race) $ python3
Python 3.7.1 (default, Nov 28 2018, 11:51:54)
[Clang 10.0.0 (clang-1000.11.45.5)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> def x(t):
...   return t
...
>>> class Foo:
...   x = x
...
>>> x_impl = Foo.x
>>> type(x_impl)
<class 'function'>```

somehow py2 treats as instance method while py3 treats it as function. Good find.  This seems to be related to https://stackoverflow.com/questions/3589311/get-defining-class-of-unbound-method-object-in-python-3 (unbound method in python3 is gone)

Reply:
FYI @astahlman @milton0825